src/controller.rs

```diff
@@ -38,6 +38,7 @@ pub struct FoxBox {
 pub const DEFAULT_HTTP_PORT: u16 = 3000;
 const DEFAULT_WS_PORT: u16 = 4000;
 const DEFAULT_HOSTNAME: &'static str = "::"; // ipv6 default.
+const DEFAULT_DOMAIN: &'static str = ".local";
```

Comment:
Not 100% sure that we need this constant at all, maybe it will be enough to just hardcode it in the ```format!``` call below